src/services/index.ts

```diff
@@ -57,6 +58,11 @@ const okrApi = new OKRApi({
   apiPath: `${API_ORIGIN_URL}${API_PREFIX}`,
 });
 
+const educationApi = new EducationApi({
+  http,
+  apiPath: `${API_ORIGIN_URL}${API_PREFIX}`,
```

Comment:
Remove these duplicates (`apiPath`). Extract a `defaultApiPath` variable

Reply:
UPD we already have that variable. It's named `apiPath`. Just replace all `apiPath: `${API_ORIGIN_URL}${API_PREFIX}`'s here with `apiPath`.